content/applications/services/helpdesk/overview/getting_started.rst

```diff
@@ -139,26 +190,32 @@ Follow the same steps to select, edit, or create an :guilabel:`SMS Template`.
 Assign stages to a team
 -----------------------
 
-Make a selection in the :guilabel:`Teams` field on the :guilabel:`Stages` form. More than one team
-may be selected, since the same stage(s) can be assigned to multiple teams.
-
- .. image:: getting_started/stages-settings-sharing.png
-   :align: center
-   :alt: View of stage setup emphasizing teams field
+Make a selection in the :guilabel:`Helpdesk Teams` field on the :guilabel:`Stages` form. More than
+one team may be selected, since the same stage can be assigned to multiple teams.
 
 Fold a stage
 ------------
 
-Check the :guilabel:`Folded in Kanban` box on the :guilabel:`Stages` form to display this stage as
-*folded* by default in the kanban view for this team.
+By default, stages are unfolded in the Kanban view. Tickets in an unfolded stage are visible in the
+pipeline under the stage name, and are considered *open*.
```

Comment:
```suggestion
By default, stages are unfolded in the Kanban view of either tickets dashboard: :guilabel:`My
Tickets` (:menuselection:`Helpdesk app --> Tickets --> My Tickets`) or :guilabel:`All Tickets`
(:menuselection:`Helpdesk app --> Tickets --> All Tickets`).

Tickets in an unfolded stage are visible in the pipeline under the stage name, and are considered
*open*.
```
I added this specificity here because, up until this point, I was under the impression you were referencing the Kanban view of the Stages themselves. I fully see how I was wrong assuming that now hahaha -- so, for any future reader who may possibly make the same mistake as me, I figured adding this quick explanation of *where* these stage elements can be employed in the app (on the Tickets pages) would prove helpful. Again, this lapse in comprehension is 100% on me hahaha but I *DO* think this quick mention will prove beneficial. Thanks! 👍